packages/client/codegen-api.yml

```diff
@@ -72,6 +72,8 @@ generates:
       extension: .generated.ts
       baseTypesPath: src/graphql/types.generated.ts
     plugins:
+      - add:
+          content: '// @ts-nocheck'
```

Comment:
I think there is one too much white space here as for a yml file (I know it works but it's not formated correctly, is it?)

Reply:
do you mean between the `//` and the `@ts-nocheck`?

Reply:
oh, my bad, all good, sorry
I thought it should be 

```
- add:
  content: '// @ts-nocheck'
```
but I'm wrong

maybe wrapping add in single quotes would help with consistency but whatever :) 

`- 'add':`